src/redux/team-members/action-creators.ts

```diff
@@ -12,55 +12,63 @@ import {
   SORT_TEAM_MEMBERS_ORDER
 } from '.';
 
-export const addTeamMemberFailure = error => ({
+import { ActionsInterface } from '../team-members/types';
+
+export const addTeamMemberFailure = (error: string): ActionsInterface => ({
```

Comment:
Isn't the ActionsInterface technically supposed to be named Action interface since you're describing the makeup of a single action interface instead of the collection of action interfaces?

Reply:
Agreed! I intended to refer to all actions, but this is more clear.